scala_config.bzl

```diff
@@ -10,6 +10,17 @@ def _validate_supported_scala_version(scala_major_version, scala_minor_version):
     if scala_major_version == "2.12" and int(scala_minor_version) < 1:
         fail("Scala version must be newer or equal to 2.12.1 to use compiler dependency tracking.")
 
+def _config_setting(scala_version):
+    return "\n".join([
```

Comment:
Maybe it would be cleaner if you format a multiline string instead of joining on newline.